src/main/java/io/github/moulberry/notenoughupdates/listener/RenderListener.java

```diff
@@ -928,6 +942,15 @@ public void onGuiScreenMouse(GuiScreenEvent.MouseInputEvent.Pre event) {
 						});
 					}
 				}
+			} else if (containerName.equals("Craft Item") && BetterContainers.recipeSearchStackIndex != -1 &&
+				((AccessorGuiContainer) eventGui).doIsMouseOverSlot(
+					cc.inventorySlots.get(BetterContainers.recipeSearchStackIndex),
+					mouseX,
+					mouseY
+				) &&
+				Mouse.getEventButton() >= 0) {
+				event.setCanceled(true);
+				NotEnoughUpdates.INSTANCE.openGui = new GuiEditSign(new TileEntitySign());
```

Comment:
I would prefer if we could not open a sign UI for this.

Reply:
how should I do it? since from what I saw that was the only way (might be wrong tho)